deploy/manifests/dev/us-east-2/tenant/storetheindex/assigner/kustomization.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: storetheindex
+
+resources:
+  - ../../../../../base/assigner
+  - ingress.yaml
+
+patchesStrategicMerge:
+  - deployment.yaml
+
+secretGenerator:
+  - name: identity
+    behavior: replace
+    files:
+      - identity.key=identity.key.encrypted # 12D3KooWDBjcDRQ7CKJeF9Yy3UKbriHfyETDrXKzTDB6biH3ibBd
```

Comment:
Specify `image` override with concrete image name and tag. As is the generated manifests will not correspond to any runnable containers.

Reply:
Done. Image name is `storetheindex` since that is the image I am using. Or, should I override the name of the images to "assigner"?

Reply:
nope all good